External/Plugins/ASCompletion/Completion/ASComplete.cs

```diff
@@ -3528,7 +3528,7 @@ private static ASExpr GetExpression(ScintillaControl sci, int position, bool ign
                 // string literals only allowed in sub-expressions
                 else
                 {
-                    if (braceCount == 0) // not expected: something's wrong
+                    if (braceCount == 0 && !hadDot) // not expected: something's wrong
```

Comment:
You sure this won't break anything out there? Sadly we don't have many tests for the GetExpression part yet.